python_modules/dagster/tox.ini

```diff
@@ -42,4 +42,4 @@ commands =
 
 [testenv:pylint]
 commands =
-  pylint -j0 --rcfile=../../pyproject.toml {posargs} dagster dagster_tests
+  pylint -j0 --rcfile=../../pyproject.toml {posargs} dagster dagster_tests --extension-pkg-whitelist='pydantic'
```

Comment:
can you comment as to why this is necessary

Reply:
This is because of a pylint issue, this flag [allows pylint to recognize various top-level imports](https://github.com/pydantic/pydantic/issues/1961#issuecomment-729288794) from the `pylint` module. Added a comment in the `.ini` file as well.